pkgs/os-specific/linux/vtune-sepdk/default.nix

```diff
@@ -0,0 +1,106 @@
+{
+  bash,
+  coreutils,
+  gnugrep,
+  gnused,
+  intel-oneapi-vtune,
+  kernel,
+  kmod,
+  lib,
+  makeWrapper,
+  p7zip,
+  procps,
+  python3,
+  shadow,
+  stdenv,
+  su,
+  which,
+}:
+
+let
+  binPath = lib.makeBinPath [
+    bash
+    coreutils
+    gnugrep
+    gnused
+    kmod
+    procps
+    python3
+    shadow
+    su
+    which
+  ];
+  libexecDir = "\${out}/libexec/vtune-sepdk";
+  modDestDir = "$out/lib/modules/${kernel.modDirVersion}/kernel/drivers/platform/x86";
+  version_majmin = builtins.head (
+    builtins.match "([0-9]+\.[0-9]+)\.[0-9]+" intel-oneapi-vtune.version
```

Comment:
Can't this use lib.versions.majorMinor? Same question for other similar usages.

Reply:
Fixed.

---
For the reference, I've took this (anti-)pattern from `libcutl`.
https://github.com/NixOS/nixpkgs/blob/79723dbb2ad738817e2a1a52d2954def4bb1f004/pkgs/development/libraries/libcutl/default.nix#L24
I've fixed it for `libcutl` as well: #332257.